enhancements/operator-hub-check.md

```diff
@@ -0,0 +1,217 @@
+
+However, would be expected following good practices where the types of checks are properly encapsulate and can easily keep maintained and extracted if required.
+
+**2. Are ONlY these checks enough to ensure that the operator bundles work well as expected?** 
+
+Definitely not. The operator bundle manifests has a lot of information which could be used by *"linters"* checks as this one with the purpose of avoid issues scenarios and give a fast response for the operator authors and jobs.
+
+This proposal only address some priority needs which are; 
+- check and notify the main recommendations for us at the moment; [channel naming convention][channel-names-doc] and operator name versioning.
+- avoid the issues with the most critical scenarios related to K8S API deprecations which will cause high impact after 1.22 release out.
+
+**3. What are the changes proposed here which affects SDK?** 
+
+This proposal is to define new checks for a Validator which is used and provided by SDK already. Then, for users are able to use its new checks SDK requires only bump the version of [operator-framework/api][oper-api] and add a new option flag `--optional-values` to get and pass options values to the Validator. 
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+
+## Summary
+
+`OperatorHubValidator` checks if the operator is respecting the required criteria to be published in the [OperatorHub.io](https://operatorhub.io/). This proposal describes criteria to be added to it. See [here](https://github.com/operator-framework/api/blob/v0.6.0/pkg/validation/internal/operatorhub.go) its checks code implementation.
+
+Many tools and projects use `OperatorHubValidator` already. Note that the new checks described by this proposal will be available via SDK:
+
+```sh
+$ operator-sdk bundle validate --help
+...
+To validate a bundle against operatorhub.io requirements specifically, in addition to the standard bundle validation:
+   
+$ operator-sdk bundle validate ./bundle --select-optional name=operatorhub
+``` 
+
+The new validator also needs to allow the checks occurs to verify the operator bundle configuration for a specific Kubernetes version where it is intend to be published:
+
+```sh
+$ operator-sdk bundle validate --help
+...
+To validate a bundle against operatorhub.io requirements specifically, in addition to the standard bundle validation for an Kubernetes version where it is intended to be distribute:
+   
+$ operator-sdk bundle validate ./bundle \
+    --select-optional name=operatorhub \
+    --optional-values="k8s=1.22"
+```
```

Comment:
```suggestion
```sh
$ operator-sdk bundle validate ./bundle \
    --select-optional name=operatorhub \
    --optional-values="k8s=1.22"
```
```

Reply:
Is not better we leave the help description to clarifies why/how it will work as well?